context/index.js

```diff
@@ -63,7 +63,16 @@ const publicPath = skuConfig.publicPath.endsWith('/')
   ? skuConfig.publicPath
   : `${skuConfig.publicPath}/`;
 
+const devServerMiddleware =
+  skuConfig.devServerMiddleware &&
+  getPathFromCwd(skuConfig.devServerMiddleware);
+
+const useDevServerMiddleware = devServerMiddleware
```

Comment:
Maybe we should throw an error if the dev middleware file doesn't exist? Otherwise people might be confused why it's not working.

Reply:
Where's the best place to throw an error?

```
if (devServerMiddleware && fs.existsSync(devServerMiddleware) === false) {
    throw new Error(`${devServerMiddleware} does not exist. Please create it and try again.`);
}
```

Reply:
The context file is fine. This seems fine to me 👍